master/getting-started/docker/index.md

```diff
@@ -32,15 +32,20 @@ you need to install and use Calico.
 
 ## Tutorials
 
-#### [Simple Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/simple-policy)
+#### [Simple Profile Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/simple-profile-policy)
 
 The above guide demonstrates Calico connectivity cross host, and how to limit
 that connectivity using simple Calico policy.
 
-#### [Advanced Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/advanced-policy)
+#### [Advanced Profile Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/advanced-profile-policy)
 
 The above guide digs deeper into advanced policy configurations for workloads.
```

Comment:
I think this needs to be expanded now to indicate that it uses a per-network profile to apply per-network sets of labels which can then be used to apply Calico label-based policy.

Perhaps not quite worded like that though :-/